.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "art"]
+	path = art
+	url = ../art
```

Comment:
This should probably be the github url?

Reply:
I believe submodule URLs are relative to the parent repository's URL, so this should pull `art` from `https://github.com/sourcebots/art` regardless of where I've checked `barcode-gen` out to on my computer.

See [documentation of `git submodule add`](https://git-scm.com/docs/git-submodule).

FWIW if this repo was mirrored elsewhere (e.g. `bitbucket.org/sourcebots/barcode-gen`) then it should automatically use the sibling URL `bitbucket.org/sourcebots/art` without having to change `.gitmodules`.

(This feature was useful when we forked all the power/motor board repositories; none of the inter-dependencies between them broke despite the canonical URLs changing from SR-land to Github because of the use of relative URLs.)

Reply:
Can you add to the readme that we need to run `git submodule init`?

Reply:
Yep, will do.